### PR TITLE
Changes parallax layers + planets for each new sector

### PR DESCRIPTION
**Pull request by TMTIME**
~~i accidentally deleted this pr while doing a fix for an upstream merge oof~~

:cl: TMTIME
add: Parallax planets and the topmost layer can now be changed for each new sector
add: Parallax layer for FTL transit
imageadd: Bad coder sprites for planets (Only added Oshan, reusing our old planets) and a few background space layers
/:cl:

[why]: # 

it look good
https://drive.google.com/file/d/1QBLarkxAQA5pxFGR5uv0pnGZ1yT9nNWq/view?usp=sharing

At some point in another PR, I will add an extra parallax layer, so we can show enemy ships in the background. This will need be to do some upstream stuff first

anyway where me shirt

### CODE REVIEW

**Comment by Dennok** [comment]
We can show a space station too.

**Comment by Qustinnus** [comment]
This looks good.

**Comment by Qustinnus** [comment]
This looks good.

File: ftl/code/datums/sector.dm [hunk 1]
```diff
@@ -32,6 +34,8 @@
 
   if(prob(station_prob))
     station = new /datum/station
+    
+  parallax_icon = pickweight(list("empty" = 3, "fog" = 1, "rocks" = 1)) //temp to prove it works
```
Comment (Qustinnus) [inline ftl/code/datums/sector.dm]: rocks should later on be linked to whether or not we have asteroids in orbit or if we're in an event. this is fine for now though.

**Comment by Dennok** [comment]
the name of branch "breaks_parallax" real breaks_parallax for low settings of paralax. On low it show white background. Need to show black.

**Comment by Dennok** [comment]
also i have planet multiple times fly away.

**Comment by Qustinnus** [comment]
ah yeah I forgot about the settings for parallax, is the planets hting for every setting or?

@TMTIME 

also make an issue report of this if you can please @Dennok

**Comment by Qustinnus** [comment]
also dennok whats your discord username

**Comment by Dennok** [comment]
i think dennok.
Planet settings defoult to TG planet settings, i think.

**Comment by Qustinnus** [comment]
i cant see you in the ftl discord, can you @ me in there

**Comment by Dennok** [comment]
i dont have ftl discord

**Comment by Qustinnus** [comment]
😨 https://discord.me/ftl13/

**Comment by TMTIME** [comment]
Yeah, low parallax will need me to sort stuff out upstream so we can change layer counts + settings

**Comment by TMTIME** [comment]
also planets looping multiple times will be down to the timing of swapping out parallax icons. I'd need to test it on worse hardware to get a good timing

**Comment by Qustinnus** [comment]
did you look at monsters implementation

**Pull request merged**